tests/e2e/e2e_test.go

```diff
@@ -19,3 +26,200 @@ func (s *IntegrationTestSuite) TestUpgrade() {
 	}
 	s.T().Logf("SUCCESS")
 }
+
+func (s *IntegrationTestSuite) TestCLITxs() {
+	mainBranch := upgrade.VersionConfig{
+		ImageTag:  "main",
+		ImageName: "evmos",
+	}
+
+	s.runInitialNode(mainBranch, repoDockerFile)
+
+	testCases := []struct {
+		name      string
+		cmd       func() (string, error)
+		expPass   bool
+		expErrMsg string
+	}{
+		{
+			name: "fail - submit upgrade proposal, no fees & insufficient gas",
+			cmd: func() (string, error) {
+				return s.upgradeManager.CreateSubmitProposalExec(
+					"v11.0.0",
+					s.upgradeParams.ChainID,
+					5000,
+					true,
+					"--fees=",
+					"--gas=50000",
+				)
+			},
+			expPass:   false,
+			expErrMsg: "insufficient fee", // TODO when the PR https://github.com/evmos/evmos/pull/1386 is merged, this may fail with out of gas error
+		},
+		{
+			name: "fail - submit upgrade proposal, insufficient fees",
+			cmd: func() (string, error) {
+				return s.upgradeManager.CreateSubmitProposalExec(
+					"v11.0.0",
+					s.upgradeParams.ChainID,
+					5000,
+					true,
+					"--fees=10aevmos",
+					"--gas=50000",
+				)
+			},
+			expPass:   false,
+			expErrMsg: "insufficient fee",
+		},
+		{
+			name: "fail - submit upgrade proposal, insufficient gas",
+			cmd: func() (string, error) {
+				return s.upgradeManager.CreateSubmitProposalExec(
+					"v11.0.0",
+					s.upgradeParams.ChainID,
+					5000,
+					true,
+					"--fees=500000000000aevmos",
+					"--gas=1000",
+				)
+			},
+			expPass:   false,
+			expErrMsg: "out of gas",
+		},
+		{
+			name: "success - submit upgrade proposal, defined fees & gas",
+			cmd: func() (string, error) {
+				return s.upgradeManager.CreateSubmitProposalExec(
+					"v11.0.0",
+					s.upgradeParams.ChainID,
+					5000,
+					true,
+					"--fees=10000000000000000aevmos",
+					"--gas=1500000",
+				)
+			},
+			expPass: true,
+		},
+		{
+			name: "success - submit upgrade proposal, using gas & gas-prices",
+			cmd: func() (string, error) {
+				return s.upgradeManager.CreateSubmitProposalExec(
+					"v11.0.0",
+					s.upgradeParams.ChainID,
+					5000,
+					true,
+					"--gas-prices=1000000000aevmos",
+					"--gas=1500000",
+				)
+			},
+			expPass: true,
+		},
+		// TODO uncomment this test when the PR https://github.com/evmos/evmos/pull/1386 is merged
+		// {
+		// 	name: "success - submit upgrade proposal, no fees (defaults to requiredFees) & sufficient gas",
+		// 	cmd: func() (string, error) {
+		// 		return s.upgradeManager.CreateSubmitProposalExec(
+		// 			"v11.0.0",
+		// 			s.upgradeParams.ChainID,
+		// 			5000,
+		// 			true,
+		// 			"--fees=",
+		// 			"--gas=1500000",
```

Comment:
this UX is very confusing, shouldn't it be something like
```suggestion
		// 			"--fees=",
		// 			"--gas=1500000",
		// 			"--gas-prices=auto",
```

Reply:
Updated. As fees auto is the default value, this will fail the validation